### ISSUE TITLE
Accessibility - Submitted Screen - TalkBack Reading Four Paragraphs as One Group of Code

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/OCIO-DOLSafety-Android#66]
For ease of reading for a TalkBack user the content in the main content section should be four separate paragraphs instead of one block of text. If a user misses something like the incident number instead of swiping back one line/paragraph they have to list to the entire block of text again. The issue happens for both anonymous and not anonymous incidents
